openwisp_controller/config/api/download_views.py

```diff
@@ -0,0 +1,42 @@
+from rest_framework.generics import RetrieveAPIView
```

Comment:
Let's add a comment on top of this file, eg:

```suggestion
# API download views are stored separately from the main API view file
# to avoid import issues when importing the main views from other modules,
# eg: OpenWISP Monitoring. Find out more information at
# https://github.com/openwisp/openwisp-monitoring/pull/480#issuecomment-1475240768

from rest_framework.generics import RetrieveAPIView
```

Some people prefer to store this info in the commit message, leaving the code "cleaner",
but if I want to quickly remember why we decided to store these views separately I'd
have to spend time to find this info in the commit history, so I prefer to leave these
comments directly in the code to save time.